collectors/ebpf.plugin/ebpf_process.c

```diff
@@ -786,11 +795,12 @@ static void ebpf_create_apps_charts(ebpf_module_t *em, struct target *root)
     if (!newly_added)
         return;
 
-    if (ebpf_modules[EBPF_MODULE_PROCESS_IDX].apps_charts)
-        ebpf_process_create_apps_charts(em, root);
-
-    if (ebpf_modules[EBPF_MODULE_SOCKET_IDX].apps_charts)
-        ebpf_socket_create_apps_charts(NULL, root);
+    int counter;
+    for (counter = 0; ebpf_modules[counter].thread_name; counter++) {
+        ebpf_module_t *ptr = &ebpf_modules[counter];
```

Comment:
Wouldn't it be better to use another name for the variable to avoid confusion with the second parameter?

Reply:
I renamed the variable, but this specific function `ebpf_create_apps_charts` does not have this variable, I think you confused with the function above it (`ebpf_process_create_apps_charts`).

Reply:
Ah, yeah, I was confused by a gap in the diff 😁. Anyway, I think it is better now.